modules/home.php

```diff
@@ -1,8 +1,36 @@
-<!-- MAIN content -->
 <div id="main">
     <div id="main-content">
-        <h3> The main content go here</h3>
+    <?php if ($user === false) { ?>
+        <h3>This is home page.</h3>
+    <?php } else { ?>
+        <h3>Users List </h3>
+        <?php
+            try {
+                $result = $mysql->query($sql);
```

Comment:
You execute an sql `$sql` without defining. You are used `$sql` in Header. Please define SQL to get users list.